rollup/internal/controller/blob_uploader/blob_uploader.go

```diff
@@ -242,10 +242,12 @@ func (b *BlobUploader) GetFirstUnuploadedBatchByPlatform(ctx context.Context, st
 		break
 	}
 
-	if len(batch.CommitTxHash) == 0 {
-		log.Debug("got batch not committed for blob uploading", "batch_index", batchIndex, "platform", platform.String())
-		return nil, nil
-	}
```

Comment:
Can we check logs in grafana to see how often we experience this case. This should give us an idea of how many blobs we will upload that won't actually be posted on L1